src/common/meta/src/key.rs

```diff
@@ -284,6 +284,7 @@ impl<T: Serialize + DeserializeOwned + TableMetaValue> DeserializedValueWithByte
     }
 
     /// Notes: used for test purpose.
+    #[cfg(any(test, feature = "testing"))]
```

Comment:
```suggestion
    #[cfg(any(test, feature = "testing"))]
```

Code is much more reliable than comment. Yes :D